src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/lettertoprison/LetterToPrisonContextFactoryTest.kt

```diff
@@ -71,13 +80,24 @@ class LetterToPrisonContextFactoryTest {
       result,
       equalTo(
         LetterToPrisonContext(
-          recall,
           FullName("Barrie Badger"),
           currentPrisonName,
           lastReleasePrisonName,
           recallDescription,
-          createdByUserDetails,
-          LocalDate.now()
+          "ABC123",
+          LocalDate.now(),
+          FullName("Casey Caseworker"),
+          LocalDate.now(),
+          NomsNumber("AA1234A"),
+          false,
```

Comment:
Ditto: How about local vals for random values of these booleans, enums and dates, so that the assert is then more convincing?